routes/users.js

```diff
@@ -81,7 +81,7 @@ router.post(
 );
 router.post(
   '/blockuser-v2',
-  auth.isAuth,
+  auth.isAuthV2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The authentication middleware has been updated from `isAuth` to `isAuthV2`. Ensure that `isAuthV2` is thoroughly tested and handles all edge cases that `isAuth` did. Also, make sure that the clients of this API are ready to handle any changes in the response or behavior due to this update.
```diff
-  auth.isAuth,
+  auth.isAuthV2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->